### DISCUSSION TITLE
[@types/chrome] chrome.cookies.SetDetails missing `partitionKey?: { topLevelSite?: string, hasCrossSiteAncestor?: boolean }`

### DISCUSSION THREAD

**Discussion by crunk1** [DefinitelyTyped/DefinitelyTyped#70669] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a1410571f3d65d7631d03ecae94ad53eeb34f147/types/chrome/index.d.ts#L1873

[`partitionKey` has been in since Chromium 119](https://developer.chrome.com/docs/extensions/reference/api/cookies#type-CookiePartitionKey)

[a comment by an automated account is not shown]

**Comment by erwanjugand** [comment]
Relates to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70502
